maven/impl-maven-embedded/src/test/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/ConfigurationStageTestCase.java

```diff
@@ -81,12 +80,13 @@ public void runTest() {
         assertEquals(shellEnvironments, invocationRequest.getShellEnvironments());
         assertEquals(threads, invocationRequest.getThreads());
         assertEquals(toolChainsFile, invocationRequest.getToolchainsFile());
+        assertEquals(globalToolChainsFile, invocationRequest.getGlobalToolchainsFile());
         assertEquals(userSettingFile, invocationRequest.getUserSettingsFile());
-        assertEquals(true, invocationRequest.isActivatedReactor());
+        assertEquals(builderId, invocationRequest.getBuilder());
```

Comment:
One handy tip here would be to use `assertj` with its `soft-assertions`. This way you would have a chance to gather all the failed conditions in one test execution rather than discovering them one after the other.